src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs

```diff
@@ -33,6 +33,16 @@ public class GenevaExporterOptions
     /// </summary>
     public IEnumerable<string>? CustomFields { get; set; }
 
+    /// <summary>
+    /// Gets or sets WithResourceAttributes.
+    ///
+    /// WithResourceAttributes specifies which resource attribute fields should be sent to Geneva.
+    ///
+    /// Any resource attributes not in WithResourceAttributes are ignored.
+    /// If WithResourceAttributes is not provided, no resource attributes will be sent to Geneva.
+    /// </summary>
+    public IEnumerable<string>? WithResourceAttributes { get; set; }
```

Comment:
Should this be -->  public  IEnumerable<string>? WithResourceAttributes { get; **init**; }

Also, `AllowedResourceAttributes` or `ResourceAttributeFilter` naming might fit better here. No strong preference will leave it to you.